STAR-Delegation/CSR-template/template-schema.json

```diff
@@ -169,7 +170,7 @@
               "minItems": 1,
               "items": {
                 "type": "string",
-                "format": "hostname"
+                "pattern": "^(((([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\\-]*[A-Za-z0-9]))|(\\*{1,2}))$"
```

Comment:
Please keep the "format", and make it an anyOf of the hostname and the two wildcards.

Reply:
done in 288ce3d